### ISSUE TITLE
cpu/cpufreq_test-server traceback when scaling_available_frequencies does not exist

### ISSUE THREAD

**Issue by bladernr** [canonical/checkbox#352]
### Bug Description

We sometimes perform cert on older machines for field engagmenets which means we need to also handle cases of older hardware having quirks. In this case, it appears that the file that the scaling test looks for does not exist, and that triggers a traceback rather than being handled gracefully.

This should be fixed to handle the traceback and provide useful feedback, and exit with a failing code so that a review is triggered when a submission is used to create a certificate.

### To Reproduce

I suspect running the cpu/cpufreq_test-server test by itself on this older hardware will trigger this.  In this case, it was an HPE DL360 G7 with Intel(R) Xeon(R) CPU X5650 @ 2.67GHz

### Environment

HPE DL360 G7
Intel(R) Xeon(R) CPU X5650 @ 2.67GHz

### Relevant log output

```shell
Unable to read file: /sys/devices/system/cpu/cpu0/cpufreq/scaling_available_frequencies
ERROR: Unable to read file: /sys/devices/system/cpu/cpu0/cpufreq/scaling_available_frequencies
Traceback (most recent call last):
  File "/tmp/nest-twac3zlm.08455310780b9ac37274c1e52d39ab1eab3e57022505975b718dd2e9b0d6a36a/cpufreq_test.py", line 138, in _read_sysfs
    with open(abs_path, 'r') as _file:
FileNotFoundError: [Errno 2] No such file or directory: '/sys/devices/system/cpu/cpu0/cpufreq/scaling_available_frequencies'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/nest-twac3zlm.08455310780b9ac37274c1e52d39ab1eab3e57022505975b718dd2e9b0d6a36a/cpufreq_test.py", line 776, in <module>
    sys.exit(main())
  File "/tmp/nest-twac3zlm.08455310780b9ac37274c1e52d39ab1eab3e57022505975b718dd2e9b0d6a36a/cpufreq_test.py", line 766, in main
    cpu_freq_test = CpuFreqTest()
  File "/tmp/nest-twac3zlm.08455310780b9ac37274c1e52d39ab1eab3e57022505975b718dd2e9b0d6a36a/cpufreq_test.py", line 114, in __init__
    scaling_freqs = self._read_sysfs(
  File "/tmp/nest-twac3zlm.08455310780b9ac37274c1e52d39ab1eab3e57022505975b718dd2e9b0d6a36a/cpufreq_test.py", line 141, in _read_sysfs
    raise CpuFreqTestError(
__main__.CpuFreqTestError
```

### Additional context

Relevant test submission: https://certification.canonical.com/hardware/202303-31340/submission/302599/
